src/figvalidationservice/src/main/java/ca/bc/gov/open/pssg/rsbc/dps/figvalidationservice/FigaroValidationImpl.java

```diff
@@ -62,5 +64,22 @@ public ValidateApplicantServiceResponse validateApplicantService(ValidateApplica
 		 
 	}
 	
+	@Override
+	/*
+	 * service method to get the response for /validateApplicantForSharing requests
+	 */
+	public ValidateApplicantForSharingResponse validateApplicantForSharing(
+			ValidateApplicantForSharingRequest validateApplicantForSharingRequest)
+			throws FigaroValidationServiceException {
+		
+		// TODO - replace the following dummy response with actual response from ORDS.  
+
+		ValidateApplicantForSharingResponse validateApplicantForSharingResponse = new ValidateApplicantForSharingResponse(
+				"N", "success", 0);
```

Comment:
@shaunmillargov do you agree we should be defining parameters that we pass here? or is this overkill.

Reply:
This specific line is only a dummy response until we get the ORDS client installed. Once it is, this line disappears. Does this answer your question @akroon3r ?

Reply:
100% thanks for clarifying.